.github/workflows/qemu-kubevirt.yaml

```diff
@@ -341,7 +350,14 @@ jobs:
           docker save -o virt-operator.tar localhost:5000/virt-operator:$EDV_VERSION
 
           rm intel-idv-kubevirt*.tar.gz
-          tar cvzf intel-idv-kubevirt-$EDV_VERSION.tar.gz *.tar *.yaml
+
+          for file in *.tar; do
+            zstd --no-progress -T0 -16 -f --long=25 "$file" -o "${file}.zst"
+          done
+
+          mkdir intel-idv-kubevirt-$EDV_VERSION
+          mv *.zst *.yaml intel-idv-kubevirt-$EDV_VERSION/
+          tar cvzf intel-idv-kubevirt-$EDV_VERSION.tar.gz intel-idv-kubevirt-$EDV_VERSION
```

Comment:
I just realized this won't quite work because the yaml files moved into the (now outdated) release directory, but the release version is different. Will write up a small patch for this - some lines need to change above.